packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -190,6 +190,24 @@ class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderEl
     }
   }
 
+  get maxFileSize(): number | undefined {
+    const maxFileSize = this.getAttribute('max-file-size');
+    return maxFileSize !== null ? parseInt(maxFileSize) : undefined;
```

Comment:
It's worth adding a radix to `parseInt`, otherwise, the parsed value could be unexpected. It could parse with a different radix depending on the string, and we probably don't want that.
```js
return maxFileSize !== null ? parseInt(maxFileSize, 10) : undefined;
```